### DISCUSSION TITLE
How to specify the remote warehouse name?

### DISCUSSION THREAD

**Discussion by misitebao** [goreleaser/goreleaser#2401] [Q&A] [answered]
I want to know how to specify the name of the remote github warehouse in the configuration file. The default is origin, but if my warehouse is set to another name, the packaging will fail!

**Comment by caarlos0** [comment, marked as answer]
there's no way of doing that right now... will create an issue out of this.

**Reply by caarlos0** [reply]
https://github.com/goreleaser/goreleaser/issues/2484
